Gutenberg/config.yml

```diff
@@ -1,14 +1,4 @@
 ref:
-  # This should have either a commit or tag key.
-  # If both are set, tag will take precedence.
-  #
-  # We have automation that reads and updates this file.
-  # Leaving commented keys is discouraged, because as the automation would ignore them and the resulting updates would be noisy
-  # If you want to use a local version, please use the LOCAL_GUTENBERG environment variable when calling CocoaPods.
-  #
-  # Example:
-  #
-  #   LOCAL_GUTENBERG=../my-gutenberg-fork bundle exec pod install
-  tag: v1.103.2
+  commit: 1b95e55629e4c9af7c0b17d14186222af0a596b9
```

Comment:
Friendly reminder to update this to the Gutenberg tag once it is available.

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/21576/commits/fb155d2be709f61980d95099f0048006dd894b3b.